src/cdkutils/cdk_utils.py

```diff
@@ -68,5 +69,6 @@ def get_secret(self, path: str) -> str:
         return secret_string
 
     def get_artifact_for_stack_output(self, pipeline: pipelines.CdkPipeline, output: core.CfnOutput) -> object:
+        # pylint: disable=no-self-use
```

Comment:
I side with pylint on this one, the warning can be addressed by doing this
```suggestion
    @staticmethod
    def get_artifact_for_stack_output(pipeline: pipelines.CdkPipeline, output: core.CfnOutput) -> object:
```